src/2XXXR0_improving_our_safety_with_a_physical_quantities_and_units_library.md

```diff
@@ -54,7 +54,7 @@ is all true, but there are many other potential users for such a library.
 
 Here is a list of some less obvious candidates:
 
-- manufacturing,
+- Manufacturing,
```

Comment:
Why does this start from the big letter? Is Manufacturing somehow special in English?

Reply:
No.
I just think this is more grammatically correct, though I don't have proof.
Alternatively, you can remove all commas at the end and removing the last item.